test/libsolidity/SolidityEndToEndTest.cpp

```diff
@@ -9615,27 +9615,6 @@ BOOST_AUTO_TEST_CASE(calling_uninitialized_function_in_detail)
 	ABI_CHECK(callContractFunction("t()"), encodeArgs());
 }
 
-BOOST_AUTO_TEST_CASE(calling_uninitialized_function_through_array)
```

Comment:
I removed this test and the one below, since they are now caught at compile time. I added them back as syntax tests, though. I don't think the tests can be adjusted as meaningful end-to-end-tests without uninitialized storage variables.

Reply:
Hm, one could consider "initializing" ``x``` with a call to
```
    function make_uninitialized_x() internal pure returns (function() internal returns (uint)[200] storage x) {
        return x;
    }
```

Reply:
I think we should keep this test. It might also be possible to change it to `function() internal returns(uint)[] memory x = new (function() itnernal returns(uint)[])(3);`